plugins/grpc/cpp/v1.65.0/buf.plugin.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/grpc/cpp
+plugin_version: v1.65.0
+source_url: https://github.com/grpc/grpc
+description: Generates C++ client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v26.1
```

Comment:
Upstream grpc is still on v26.1 - there appear to be some PRs that attempted an upgrade to v27.2 but they've since been reverted.

Reply:
https://github.com/grpc/grpc/blob/45479fda2a083c4e46310b9bb6e45e625e381269/bazel/grpc_deps.bzl#L58-L73